### PR TITLE
Cards for Wgamma_Monophoton_Ptg130_2jets , Zllgamma_Monophoton_Ptg130_2jets and Znunugamma_Monophoton_Ptg130_2jets LO samples

### PR DESCRIPTION
**Pull request by gomber**

### CODE REVIEW

**Comment by efeyazgan** [comment]
please: 
- cleanup commented inputs
- remove  p and j definitions, remove duplicated lepton definitions
- use
$DEFAULT_PDF_SETS = lhaid
$DEFAULT_PDF_MEMBERS  = reweight_PDF

Also note that we are waiting to conclude on our final check on one parameter (pdfwgt) for MLM.

**Comment by perrozzi** [comment]
fyi, pdfwgt must be switched to True

**Comment by perrozzi** [comment]
@gomber any news?

**Comment by bhawnasinp** [comment]
@perrozzi  thanks for the comments. I didn't see the replies till the point you mentioned my name, which is weird. I will commit the new cards today,

**Comment by gomber** [comment]
I have updated the cards, with the above comments , but not sure about
"cleanup commented inputs" comment.

Do you mean shall I remove all the comments from run_card.dat? files 
I personally like those comments, as those are helpful in reminding what the input parameters are.

**Comment by efeyazgan** [comment]
In the run cards for MLM, please set T = pdfwgt.

**Comment by efeyazgan** [comment]
Please remove the banners as well.

**Comment by gomber** [comment]
sorry what do you mean by banners? Statements like these : 
#*******************
# Running parameters
#*******************

**Comment by efeyazgan** [comment]
+#************************************************************
+#*                     MadGraph5_aMC@NLO                    *
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 2.2.2                 2014-11-06         *
+#*                                                          *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#*                                                          *
+#*               Command File for MadGraph5_aMC@NLO         *
+#*                                                          *
+#*     run as ./bin/mg5_aMC  filename                       *
+#*                                                          *
+#************************************************************

File: bin/MadGraph5_aMCatNLO/cards/production/2017/Wgamma_Monophoton_Ptg130_2jets/Wgamma_Monophoton_Ptg130_2jets_run_card.dat~ [hunk 1]
```diff
@@ -0,0 +1,281 @@
+#*********************************************************************
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/Wgamma_Monophoton_Ptg130_2jets/Wgamma_Monophoton_Ptg130_2jets_run_card.dat~]: this is a .dat~ file, please remove

File: bin/MadGraph5_aMCatNLO/cards/production/2017/Znunugamma_Monophoton_Ptg130_2jets/Znunugamma_Monophoton_Ptg130_2jets_run_card.dat~ [hunk 2]
```diff
@@ -0,0 +1,281 @@
+#*********************************************************************
+#                       MadGraph5_aMC@NLO                            *
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/Znunugamma_Monophoton_Ptg130_2jets/Znunugamma_Monophoton_Ptg130_2jets_run_card.dat~]: remove this file too

File: bin/MadGraph5_aMCatNLO/cards/production/2017/Zllgamma_Monophoton_Ptg130_2jets/Zllgamma_Monophoton_Ptg130_2jets_proc_card.dat [hunk 3]
```diff
@@ -0,0 +1,13 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model sm-ckm_no_b_mass
+define l+ = e+ mu+ ta+
+define l- = e- mu- ta-
+define vl = ve vm vt
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/Zllgamma_Monophoton_Ptg130_2jets/Zllgamma_Monophoton_Ptg130_2jets_proc_card.dat]: remove unused neutrino definitions

File: bin/MadGraph5_aMCatNLO/cards/production/2017/Znunugamma_Monophoton_Ptg130_2jets/Znunugamma_Monophoton_Ptg130_2jets_proc_card.dat [hunk 4]
```diff
@@ -0,0 +1,13 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model sm-ckm_no_b_mass
+define l+ = e+ mu+
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/Znunugamma_Monophoton_Ptg130_2jets/Znunugamma_Monophoton_Ptg130_2jets_proc_card.dat]: remove unused lepton definitions

File: bin/MadGraph5_aMCatNLO/cards/production/2017/Znunugamma_Monophoton_Ptg130_2jets/Znunugamma_Monophoton_Ptg130_2jets_run_card.dat [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+#
+#*********************************************************************
+# Store info for systematics studies                                 *
+# WARNING: If use_syst is T, matched Pythia output is                *
+#          meaningful ONLY if plotted taking matchscale              *
+#          reweighting into account!                                 *
+#*********************************************************************
+ T	=  use_syst       ! Enable systematics studies
+#
+#**************************************
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/Znunugamma_Monophoton_Ptg130_2jets/Znunugamma_Monophoton_Ptg130_2jets_run_card.dat]: I believe the syscalc part can be removed, right @qliphy @kdlong ?

**Comment by gomber** [comment]
@kdlong @qliphy if you confirm the above, I will go ahead and do the changes ?

**Comment by qliphy** [comment]
I agree those syscalc part can be removed.  Except this following line to be kept
   T  = use_syst      ! Enable systematics studies

The other parts on PDF sets are set in runcmsgrid_LO.sh

**Pull request merged**